projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing-helpers/ticket-listing.ts

```diff
@@ -197,3 +198,44 @@ function getIdInRow(rowNumber: number) {
     .eq(ID_COLUMN)
     .find('a');
 }
+
+export function verifyTicketIdIsSmallerInNextPageComparedToPreviousPageByComparingIds(){
+  const TOTAL_NUMBER_OF_PAGES_TO_VISIT = 3;
+  for(let page = 1; page < TOTAL_NUMBER_OF_PAGES_TO_VISIT; page++){
+    getIdInRow(FIFTH_ROW_TICKET_LIST).then((id) => {
+      const smallerId = parseInt(id.text(), 10);
+      var next_page = page+1;
+      cy.get(`aria-label="page ${next_page}"`).click();
+      getIdInRow(FIFTH_ROW_TICKET_LIST)
+        .invoke('text')
+        .then(parseFloat)
```

Comment:
Why parseFloat and not parseInt?
Or how about creating a function to parse that smalledIr and the biggerId will use

Reply:
just to be specific, do you mean extract the `.then(parseFloat)`? if we do that will we be able to compare the two values? As i thought that is not possible in cypress?

Reply:
It was confusing why on the one side you were using parseInt and in the other side parseFloat.
All occurrences with parseInt should do the work